tfac/PCA_functions.py

```diff
@@ -0,0 +1,24 @@
+"""Contains function for performing PCA on OHSU data"""
+from dataHelpers import Tensor_LINCS_MEMA
+from sklearn.preprocessing import StandardScaler
```

Comment:
the data has already been normalized in the importing function, so you don't need to do it again.